Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityToolkitFiles.cs

```diff
@@ -144,25 +93,37 @@ static MixedRealityToolkitFiles()
 
         private static void SearchForFoldersAsync(string rootPath)
         {
-            foreach (MixedRealityToolkitModuleType module in Enum.GetValues(typeof(MixedRealityToolkitModuleType)))
+            foreach (string folder in Directory.EnumerateDirectories(rootPath))
             {
-                IEnumerable<string> directories = Directory.GetDirectories(rootPath, MixedRealityToolkitDirectory(module), SearchOption.AllDirectories);
+                RegisterModuleFolder(folder);
+            }
+        }
 
-                if (directories.Count() == 0)
+        private static void RegisterModuleFolder(string folder)
```

Comment:
Consider renaming as TryRegisterModuleFolder (and similiarly as TryUnregisterModuleFolder)

Reply:
Done.